src/Semver/SemVersion.cs

```diff
@@ -129,21 +110,13 @@ public static SemVersion Parse(string version, bool strict = false)
             if (!match.Success)
                 throw new ArgumentException("Invalid version.", "version");
 
-#if NETSTANDARD
```

Comment:
@maxhauser I think you want to revert this change. I'm a bit worried about this in all of my projects because I really don't know how parsing foreign formats will work in .net standard and there has been limited docs / feedback on this.